src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -184,7 +184,7 @@ class SelectedFilesDownloadModal extends React.PureComponent {
     }
 
     render(){
-        const { onHide, filenamePrefix, selectedFiles, fileCountUnique, fileCountWithDupes, context } = this.props;
+        const { onHide, filenamePrefix, selectedFiles, fileCountUnique, fileCountWithDupes, context, session } = this.props;
```

Comment:
Codacy found an issue: ['fileCountWithDupes' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6414303)